src/mon/Monitor.h

```diff
@@ -558,6 +558,9 @@ class Monitor : public Dispatcher,
 public:
   epoch_t get_epoch();
   int get_leader() const { return leader; }
+  string get_leader_name() {
```

Comment:
@jecluis  Hello! This name just like the below get_quorum_names. If here change may the below name also need to change, that is right?